include/ipr/impl

```diff
@@ -1198,6 +1199,7 @@ namespace ipr {
          Initializer_list* make_initializer_list(const ipr::Expr_list&, Optional<ipr::Type> = {});
          impl::Id_expr* make_id_expr(const ipr::Name&, Optional<ipr::Type> = {});
          Id_expr* make_id_expr(const ipr::Decl&);
+         Label* make_label(const ipr::Identifier&, Optional<ipr::Type> = {});
          Not* make_not(const ipr::Expr&, Optional<ipr::Type> = {});
```

Comment:
It is not clear we will ever assign a type to an identifier label.  However that is an issue to be resolved at the interface level.